mne/commands/mne_freeview_bem_surfaces.py

```diff
@@ -30,6 +30,10 @@ def freeview_bem_surfaces(subject, subjects_dir, method):
     """
     subjects_dir = get_subjects_dir(subjects_dir, raise_error=True)
 
+    if (subject is None or
+            not os.path.isdir(os.path.join(subjects_dir, subject))):
+        raise ValueError("Wrong path. Check subjects-dir or subject argument.")
```

Comment:
please print the name of the dir that you expected and that was missing. It will provide better feedback.

Reply:
Hm, it's hard to tell. People can keep their mri scans wherever. So that statement catches two cases: either subject is not in that folder, or some path (to subject directory or to subject itself) was misspelled. Not sure what do you mean by "expected" path here?

Reply:
please print `os.path.isdir(os.path.join(subjects_dir, subject))` is case it's not available.

Reply:
I see, done!